assets/src/components/CommandPalette.tsx

```diff
@@ -38,8 +57,8 @@ function buildActions(applications, nav) {
         app,
         icon: hasIcons(app) ? (
           <AppIcon
-            url={getIcon(app)}
-            size="xxsmall"
+            src={getIcon(app)}
+            // size="xxsmall"
```

Comment:
Can you remove commented out code fragments (if it doesn't make sense to keep them)?

Reply:
Done.